presto-native-execution/src/test/java/com/facebook/presto/nativeworker/IcebergExternalWorkerQueryRunner.java

```diff
@@ -29,12 +29,12 @@ public static void main(String[] args)
         Logging.initialize();
 
         // Create tables before launching distributed runner.
-        QueryRunner javaQueryRunner = PrestoNativeQueryRunnerUtils.createJavaIcebergQueryRunner(true);
+        QueryRunner javaQueryRunner = PrestoNativeQueryRunnerUtils.createJavaIcebergQueryRunner(false);
         NativeQueryRunnerUtils.createAllIcebergTables(javaQueryRunner);
         javaQueryRunner.close();
 
         // Launch distributed runner.
-        DistributedQueryRunner queryRunner = (DistributedQueryRunner) PrestoNativeQueryRunnerUtils.createNativeIcebergQueryRunner(true);
+        DistributedQueryRunner queryRunner = (DistributedQueryRunner) PrestoNativeQueryRunnerUtils.createNativeIcebergQueryRunner(true, false);
```

Comment:
Is there a reason why we change the parameter `addStorageFormatToPath` here from `true` to `false`? As I understand, the original behavior in native worker `IcebergExternalWorkerQueryRunner` would always add storage format to data dictionary path, perhaps in order to distinguish from java worker. So should we follow the original behavior? Any misunderstanding please let me know.

Reply:
Thanks for you review @hantangwangd  
So the idea here is to keep default value for  `addStorageFormatToPath` in IcebergExternalWorkerQueryRunner which would generate default `PARQUET` data and user if wish to tweak and generate different format they can tweak this flag here accordingly and data directory location would be tweaked accordingly.

Reply:
Thanks for the explanation, I understand the motivation behind adding this parameter here. What I meant is that should we set the value of `addStorageFormatToPath` to `true` in line 32 and 37 to follow the original default behavior for native worker?

Reply:
Thanks for the details. 
One other point here is, Java IcebergQueryRunner by default doesn't include StorageFormatToPath in the final catalog directory, so in default scenario if data is created from there and then IcebergExternalWorkerQueryRunner is started to reuse existing data it will not be the case if we make `addStorageFormatToPath` to `true`

Reply:
When you mentioned the Java `IcebergQueryRunner` that created the data, do you refer to the `javaQueryRunner` created in line 32? If so, the original default behavior is setting `StorageFormatToPath` true when creating it:

```
QueryRunner javaQueryRunner = PrestoNativeQueryRunnerUtils.createJavaIcebergQueryRunner(true);
```

So the following native `IcebergExternalWorkerQueryRunner` instance can reuse the data successfully. Or did I misunderstand you meaning?

Reply:
I am sorry about the confusion @hantangwangd . With Java `IcebergQueryRunner` I meant the one from Java setup here - https://github.com/prestodb/presto/blob/master/presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergQueryRunner.java#L251

Reply:
Thank you for the information @agrawalreetika. I'm not so familiar with test cases for native worker, so one last question to clarify the situation. Due to the fact that `IcebergExternalWorkerQueryRunner` is used in tests, do we currently have a test scenario as you mentioned above that may lead in trouble when `IcebergExternalWorkerQueryRunner` trying to reuse the data generated by `IcebergQueryRunner` with default behavior? And If so, could the test scenario work well and pass before this change?

Reply:
> Is there a reason why we change the parameter `addStorageFormatToPath` here from `true` to `false`? As I understand, the original behavior in native worker `IcebergExternalWorkerQueryRunner` would always add storage format to data dictionary path, perhaps in order to distinguish from java worker. So should we follow the original behavior? Any misunderstanding please let me know.

@hantangwangd I believe the "true" setting of addStorageFormatToPath in native Hive query runner was a mistake. Several reasons:
1. Java HiveQueryRunner does not add the file format to the path. Therefore the data created by Java HiveQueryRunner cannot be seen by HiveExternalQueryRunner by default and vice versa. The same problem is now happening on Iceberg query runners and that's why this fix.
2. We were unable to use both file formats at the same time if the file format was added to path. A catalog may contain databases with different file formats, and a schema(database) may have tables with different file formats. With a file format string added in the middle of the path, it's impossible for you to see them at the same time. 

Based on these deficiencies, we introduced the `addStorageFormatToPath` variable, just to provide some backward compatibility for people who used it. I believe the file forat should be removed from the path in the future.

Reply:
> 2. We were unable to use both file formats at the same time if the file format was added to path. A catalog may contain databases with different file formats, and a schema(database) may have tables with different file formats. With a file format string added in the middle of the path, it's impossible for you to see them at the same time.

Oh, yes, that's a huge deficiency I did not aware of.

> Based on these deficiencies, we introduced the `addStorageFormatToPath` variable, just to provide some backward compatibility for people who used it. I believe the file forat should be removed from the path in the future.

Thanks for the explanation, I got the point why we changed the default value here, that makes sense.